phpBB/phpbb/extension/metadata_manager.php

```diff
@@ -93,29 +93,18 @@ public function get_metadata($element = 'all')
 		{
 			case 'all':
 			default:
-				// Validate the metadata
-				if (!$this->validate())
-				{
-					return false;
-				}
-
+				$this->validate();
```

Comment:
All the changes in this file should already be included, as there are no changes from 3.1 to 3.2.  But they show up as different from the current 3.2 branch...